### PR TITLE
Fix `dump_suspended_threads`

### PR DESCRIPTION
**Pull request by msimberg**
Follow-up to #658.

### CODE REVIEW

**Comment by msimberg** [comment]
bors merge

File: libs/pika/schedulers/include/pika/schedulers/queue_helpers.hpp [hunk 1]
```diff
@@ -47,7 +47,7 @@ namespace pika::threads::detail {
             return false;
 
         // attempt to output possibly deadlocked threads occasionally only
-        if (PIKA_LIKELY((idle_loop_count++ % PIKA_IDLE_LOOP_COUNT_MAX) != 0))
+        if (PIKA_LIKELY((idle_loop_count % PIKA_IDLE_LOOP_COUNT_MAX) != 0))
```
Comment (biddisco) [inline libs/pika/schedulers/include/pika/schedulers/queue_helpers.hpp:50]: good catch

**Review by biddisco — APPROVED** [review]
Seems reasonable

**Comment by msimberg** [comment]
bors merge

**Pull request merged**